modules/nw-ovn-kubernetes-rollback-live.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-kubernetes-rollback-live_{context}"]
+= Using the live migration method to roll back to the OpenShift SDN network plugin
+
+As a cluster administrator, you can rollback to the OpenShift SDN Container Network Interface (CNI) network plugin by using the live migration method. During the migration with this method, nodes are automatically rebooted and service to the cluster is not interrupted.
```

Comment:
```suggestion
As a cluster administrator, you can roll back to the OpenShift SDN Container Network Interface (CNI) network plugin by using the live migration method. During the rollback with this method, nodes are automatically rebooted and service to the cluster is not interrupted.
```

Reply:
I change all of these (there were four). I think it was always "rollback" previously and I just copied it over. Fixed now though